jboss-kie-wildfly-common/added/launch/jboss-kie-wildfly-security-login-modules.sh

```diff
@@ -66,12 +67,16 @@ function configure_ldap_login_module() {
         log_info "AUTH_LDAP_URL not set. Skipping LDAP integration..."
         return
     fi
-    log_info "AUTH_LDAP_URL is set to ${AUTH_LDAP_URL}. Added LdapExtended login-module"
-
+    log_info "AUTH_LDAP_URL is set to ${AUTH_LDAP_URL}. Added LdapExtended login-module" 
+        
     # RHPAM-1422, if the RealmDirect is set as Required, ldap auth will fail.
     # TODO remove it out as part of the CLOUD-2750
     sed -i 's|<login-module code="RealmDirect" flag="required">|<login-module code="RealmDirect" flag="optional">|' "${CONFIG_FILE}"
-    local login_module=$(build_login_module "LdapExtended" "required")
+    if [ "${AUTH_LDAP_LOGIN_MODULE}" = "optional" ]; then
```

Comment:
why not just:

```
local ldap_login_module_flag="${AUTH_LDAP_LOGIN_MODULE:-required}"
if [ "${ldap_login_module_flag}" = "optional" ]; then
    ldap_login_module_flag="optional"
fi

local login_module=$(build_login_module "LdapExtended" "${ldap_login_module_flag}")
```

there's no need to replicate the entire line since only one parameter is being changed.